pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -1,45 +1,61 @@
+## Introducing uswds-compile
+
+We’ve developed `uswds-compile`, a tool [hosted on GitHub](https://github.com/uswds/uswds-compile), to help teams compile USWDS Sass. This tool provides simple Gulp 4 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
+
+We recommend using `uswds-compile` as a quick way to get up and running with USWDS Sass compilation.
 
 {: .site-note }
-**Note:** If you want `uswds-compile` to set up a Sass entry point for you, skip directly to [Step 2](#step-2-install-uswds-compile). We provide Step 1 for context: why you need a Sass entry point, what it needs to include, and how to set one up manually.
+Note: One of the benefits of using `uswds-compile` is that it will set up a Sass entry point for you. If you plan on using this tool, you can safely skip the next section about Sass entry points and jump straight to the [step-by-step instructions](#step-1-install-uswds-compile) that explain how to install and use `uswds-compile`.
+
+If you prefer to set up Sass compilation yourself, the next thing you'll need to do is set up (or edit) your Sass entry point.
+
+## About Sass entry points
+
+A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase. Often, a project's Sass entry point is named something like `index.scss` or `styles.scss`.
+
+### Setting up USWDS in your Sass entry point
 
-## Step 1: Set up your project’s Sass entry point
+If your project does not yet have a Sass entry point, create a file called `index.scss` or `styles.scss`.
 
-A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase.
+When your entry point is ready, you'll need to load USWDS Sass. Your Sass entry point will need to do the following three tasks in this order:
 
-Your project’s Sass entry point is a simple file that only needs to do the following three tasks:
-1. Include a USWDS [settings file]({{ site.baseurl }}/documentation/settings/)
-2. Include the USWDS source code
-3. Include your project’s custom Sass
+1. Define [USWDS settings]({{ site.baseurl }}/documentation/settings/)
+2. Load  USWDS source code
+3. Load your project’s custom Sass
```

Comment:
It is an interesting idea, for sure. My viewpoint in approaching this is that the  "Getting started for developers" guide is something that anyone familiar with web development at all should be able to complete, regardless of whether they have skill with Sass or not. It would be interesting to define the minimum skillsets we expect out of users though, even if it is for internal use only to start.